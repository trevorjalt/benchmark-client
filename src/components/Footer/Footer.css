.Footer {
    height: 10vh;
    display: flex;
    align-items: center;
    justify-content: center;
    background-color: #FFD23F;
    margin-top: 4vh;
    position: absolute; 
    left: 0 ; right: 0; bottom: 0;
}