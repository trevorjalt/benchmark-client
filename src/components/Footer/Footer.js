import React, { Component } from 'react'
import './Footer.css'

export default class Footer extends Component {
    render() {
        return (
            <div className='Footer'>
                <h4 className= "footer-copy">Â© Copyright Trevor J Alt. All Rights Reserved.</h4>
            </div>
        )
    }
}