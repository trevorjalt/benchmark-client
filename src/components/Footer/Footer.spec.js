import React from 'react'
import Footer from './Footer'
import toJson from 'enzyme-to-json'
import { shallow } from 'enzyme'

describe(`Footer Component`, () => {
    it (`renders without crashing`, () => {
        const wrapper = 
            shallow (
                <Footer />
            )
        expect(toJson(wrapper)).toMatchSnapshot()
    })
})