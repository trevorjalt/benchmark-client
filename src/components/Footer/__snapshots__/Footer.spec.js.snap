// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer Component renders without crashing 1`] = `
<div
  className="Footer"
>
  <h4
    className="footer-copy"
  >
    Â© Copyright Trevor J Alt. All Rights Reserved.
  </h4>
</div>
`;
